### PR TITLE
Prepare 2.53.0 release

### PR DESCRIPTION
**Pull request by krajorama**

### CODE REVIEW

**Comment by SuperQ** [comment]
Running a load test, mostly to test that I didn't break the test-infra with some Python updates.

/prombench v2.52.0

**Comment by SuperQ** [comment]
Boo, I think I broke the comment monitor job :(

**Review by ArthurSens — APPROVED** [review]
\

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,20 +2,13 @@
...
-* [BUGFIX] Rules: Fix Group.Equals() to take in account the new queryOffset too. Followup to #14061. #14273
-
-## 2.53.0-rc.0 / 2024-06-06
-
-This release changes the default for GOGC, the Go runtime control for the trade-off between excess memory use and CPU usage. We have found that Prometheus operates with minimal additional CPU usage, but greatly reduced memory by adjusting the upstream Go default from 100 to 50.
-
 * [CHANGE] Rules: Execute 1 query instead of N (where N is the number of alerts within alert rule) when restoring alerts. #13980 #14048
-* [CHANGE] Runtime: Change GOGC threshold from 100 to 50 #14176
-* [FEATURE] Rules: Add new option `query_offset` for each rule group via rule group configuration file and `rule_query_offset` as part of the global configuration to have more resilience for remote write delays. #14061 #14216
+* [CHANGE] Runtime: Change GOGC threshold from 100 to 50 #14176 #14285
```
Comment (roidelapluie) [inline CHANGELOG.md:10]:
```suggestion
* [CHANGE] Runtime: Change GOGC threshold from 100 to 75 #14176 #14285
```

**Pull request merged**